Source/WebGPU/WebGPU/Device.mm

```diff
@@ -43,11 +43,48 @@
 #import "SwapChain.h"
 #import "Texture.h"
 #import <algorithm>
+#import <notify.h>
 #import <wtf/StdLibExtras.h>
 #import <wtf/WeakPtr.h>
 
 namespace WebGPU {
 
+static bool frameCaptureEnabled(std::optional<bool> enable = std::nullopt)
```

Comment:
I wonder if it would be better to have this either:

- as a property on Device, to avoid having `frameCaptureEnabled` being both a getter and a setter, or
- with the `gpuFrameCaptureEnabled` variable outside, and a separate method for the getter and setter.

But not a big deal to me.

Reply:
Good idea @grorg , I’ll use two class properties instead.